Update tutorial now that 24.04 is out.md (#629)
open-spaced-repetition/fsrs4anki: docs/tutorial.md

### Maximum interval

> The  Maximum interval setting works the same way as when using the default algorithm. It is the maximum number of days that a card can wait until it is shown again. For more information, see [Maximum interval](https://docs.ankiweb.net/deck-options.html#maximum-interval) in the Anki manual. In Anki 24.04, it is located in the "Advanced" section when FSRS is enabled, rather than in the "FSRS" section.

Author: Why "even in"?

> The  Maximum interval setting works the same way as when using the default algorithm. It is the maximum number of days that a card can wait until it is shown again. For more information, see [Maximum interval](https://docs.ankiweb.net/deck-options.html#maximum-interval) in the Anki manual. In Anki 24.04, it is located in the "Advanced" section when FSRS is enabled, rather than in the "FSRS" section.

Reviewer: Because the setting was in the Advanced section in 23.10 and 23.12 too.

> The  Maximum interval setting works the same way as when using the default algorithm. It is the maximum number of days that a card can wait until it is shown again. For more information, see [Maximum interval](https://docs.ankiweb.net/deck-options.html#maximum-interval) in the Anki manual. In Anki 24.04, it is located in the "Advanced" section when FSRS is enabled, rather than in the "FSRS" section.

Author: Ah, my bad. I guess we can remove this sentence entirely. Same for "Custom scheduling", just remove that sentece. For some reason I thought it was under a different section before.

> ### SM-2 retention

Reviewer: ```suggestion
### Historical retention (SM-2 retention)
```

> "SM-2 retention" refers to your average retention before you started using FSRS.

Reviewer: ```suggestion
"Historical retention" (called "SM-2 retention" before Anki 24.04) is the average retention in the past.
```

> You need to configure this value only if you have cards with missing or incomplete review logs. Since review logs typically won't be missing unless you have explicitly deleted them to free up space or you have used some add-ons that modify the review history, **most users will not need to adjust this value**. In Anki 24.04, it's called Historical Retention, and it is located in the "Advanced" section when FSRS is enabled rather than in the "FSRS" section.

Reviewer: ```suggestion

When some of your review history is missing, FSRS needs to fill in the gaps. By default, it will assume that when you did those old reviews, you remembered 90% of the material. If your old retention was significantly higher or lower than 90%, adjusting this option will allow FSRS to better approximate the missing reviews. Your review history may be incomplete for two reasons:
1. Because you've used the 'ignore reviews before' option.
2. Because you've previously deleted review logs to free up space, used some add-ons that modify the review history, or imported material from a different SRS program. 
The latter is quite rare, so unless you've used the 'ignore reviews before' option, you probably don't need to adjust this setting.
Even in Anki 24.04, it is located in the "Advanced" section, not in the "FSRS" section.
```

![image](https://github.com/open-spaced-repetition/fsrs4anki/assets/32575846/fe61aaa6-cecb-4476-9ed2-9db05b63c7de)

### Ignore reviews before

> In Anki 24.04, a feature to exclude review history until a specific date from optimization has been added. Only reviews after the selected date will be used by the optimizer, as well as during evaluation.

Reviewer: ```suggestion
This is a new feature added in Anki 24.04. If set, review history up to the provided date will be ignored when optimizing and evaluating FSRS parameters. 
```

![image](https://github.com/open-spaced-repetition/fsrs4anki/assets/83031600/1d53d964-2136-43b2-977f-5116141c38f9)

> It is useful if you have been misusing Hard. Hard should be used as a passing grade, not a failing grade. You can also use this feature if your learning habits have changed very significantly over time, however, FSRS can likely adapt to those changes.

Reviewer: ```suggestion

This can be useful if you imported someone else's scheduling data, or if you have been misusing Hard. Hard should be used as a passing grade, not a failing grade. This feature can also be helpful if your learning or rating habits have changed significantly over time.
```

Since the tool is experimental, it is better to use your intuition to come up with a value of desired retention. However, the suggested retention can be useful as a reference when you have no idea of what you want your retention rate to be.

In Anki 24.04, this feature has been reworked. You no longer need to input "Deck size" and "Minutes study/day", only "Days to simulate". The result will also depend on the maximum interval setting. Previously, the goal was to find the value of desired retention that would allow you to remember as much as possible within the given time constraints. Now the goal is to find the values of desired retention that minimizes the workload/acquired knowledge ratio. For more details, please read this: https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-Optimal-Retention. It's important to note that you can set your desired retention higher than recommended if you want to do **more** work (minutes of studying per day) to remember **more**, but you shouldn't set your desired retention lower than recommended because you would have to do **more** work to remember **less**.

## Step 6: (optional) Custom Scheduling

> "Custom scheduling" allows you to introduce new scheduling rules on top of FSRS. This feature is only for advanced users and developers. In Anki 24.04, it is located in the "Advanced" section when FSRS is enabled, rather than in the "FSRS" section.

Reviewer: ```suggestion
"Custom scheduling" allows you to introduce new scheduling rules on top of FSRS. This feature is only for advanced users and developers. Even in Anki 24.04, it is located in the "Advanced" section, not in the "FSRS" section.
```
